validate.lic

```diff
@@ -398,6 +398,16 @@ class DRYamlValidator
             .each { |info| error("Hunt '#{info[:zone]}' has a retreat threshold set, but no ranged weapon skills to train. You may get stuck retreating and not training any weapons because enemies will not be attacked.") }
   end
 
+  def assert_that_barb_buffs_are_in_base_spells(settings)
+    return less setting.buff_nonspells['barb_buffs']
+
+    supported_buffs = get_data('spells').barb_abilities
+
+    setting.buff_nonspells['barb_buffs']
+        .select { |name| supported_buffs.value?(name) }
+        .each { |desc| error("Barb ability '#{name}' not yet supported. Please make an issue on Git for it to be added!") }
```

Comment:
Should `Git` be `GitHub`? If so, could you include a link to the repo in the message?